llvm/lib/MC/RepoObjectWriter.cpp

```diff
@@ -827,6 +839,43 @@ RepoObjectWriter::writeDebugLineHeader(TransactionType &Transaction,
   return {};
 }
 
+pstore::index::digest
+RepoObjectWriter::updateFragmentDigest(const ticketmd::DigestType &InitailHash,
+                                       FragmentContentsType &FragmentContent) {
+  MD5 FragmentHash;
+  MD5::MD5Result FragmentDigest;
+  auto IsEmptyDependent = FragmentContent.Dependents.empty();
```

Comment:
Can this be const? I'd be tempted to call it "HasDependents"

Reply:
Yes, changed to "const bool HasDependents = !FragmentContent.Dependents.empty();"